zazu/util.py

```diff
@@ -158,31 +161,38 @@ def prompt(text, default=None, expected_type=str):
     return expected_type(result)
 
 
-def pick(choices, message):
+def pick(choices, message, allow_multiple=False):
```

Comment:
I think allow_multiple here should be called allow_none? Also add it to the arguments doc list.